pallets/frequency-tx-payment/src/mock.rs

```diff
@@ -209,12 +210,14 @@ impl pallet_transaction_payment::Config for Test {
 	type OperationalFeeMultiplier = ConstU8<5>;
 }
 
+pub const TEST_TOKEN_PER_CAPACITY: u32 = 10;
```

Comment:
I made this const so it could be used in the helper function below without making it use generics.